Surface_mesh_topology/include/CGAL/Surface_mesh_topology/internal/Path_on_surface_with_rle.h

```diff
@@ -141,6 +143,17 @@ class Path_on_surface_with_rle
   #endif //CGAL_PWRLE_TURN_V2
   {}
 
+  Path_on_surface_with_rle(const Self& apath): m_MQ(apath.m_MQ),
+                                               m_path(apath.m_path),
+                                               m_is_closed(apath.m_is_closed),
+                                               m_length(apath.m_length),
+                                               m_use_only_positive(apath.m_use_only_positive),
+                                               m_use_only_negative(apath.m_use_only_negative)
+#ifdef CGAL_PWRLE_TURN_V2
+                                             , m_darts_ids(apath.m_darts_ids)
+#endif //CGAL_PWRLE_TURN_V2
+  {}
```

Comment:
Seeing the implementation of it, I think it could be:

```suggestion
  Path_on_surface_with_rle(const Path_on_surface_with_rle&) = default;
```